docs/pages/admin-guides/access-controls/idps/saml-microsoft-entra-external-id.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Access Azure Portal and CLI 
+description: Access Azure Portal and CLI by authenticating with Teleport SAML IdP
+---
+
+This guide explains how to integrate Teleport SAML IdP with [Microsoft Entra External ID](https://learn.microsoft.com/en-us/entra/external-id/)
+so that users can access Azure Portal and CLI by authenticating with Teleport.
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster version (=teleport.version=) or above. If you
+  want to get started with Teleport, [sign up](https://goteleport.com/signup)
+  for a free trial.
+- If you're new to SAML, consider reviewing our [SAML identity provider (IdP)](../../../reference/access-controls/saml-idp.mdx) 
+reference before proceeding.
+- Teleport user with permission to create a SAML service provider resource. The preset `editor` role has this permission.
+- Access to Microsoft Entra ID tenant and Azure subscription. 
+- Microsoft Entra ID user with permissions to create an identity provider, create and manage users, assign roles,
+and manage billing subscriptions. The "Global Administrator" role grants all the required permissions to follow this guide. We 
+recommend to scope down administrative permission in production.
+
+## Step 1/5. Determine if DNS update is required
+
+When you create an identity provider in Microsoft Entra External ID, you need to associate a 
+domain name with the identity provider.
+
+For example, given a user account `alice@example.com`, in order to authenticate this user with Teleport, 
+the domain `example.com` must be associated with the Teleport SAML IdP. 
+
+A domain can be associated with an IdP in two places:
+- Assign a domain name to the IdP when adding that IdP to the Microsoft Entra External ID. 
+- Assign a domain name to the IdP after the IdP is created.
+
+In both the cases, if your Teleport cluster and the user account belong to different domains, 
+you must update the DNS `TXT` record of the domain. Refer to the table below. 
+
+| Domain | User account | Teleport cluster | Domain association |
+| - | - | - | - |
+| example.com | alice@example.com | https://example.com, https://teleport.example.com | Domain can be associated without DNS update. |
+| example.com | alice@example.com | https://mytenant.teleport.sh| Domain can be associated but DNS update is required. |
```

Comment:
```suggestion
| example.com | alice@example.com | https://example.teleport.sh| Domain can be associated but DNS update is required. |
```
We're moving away from "tenant" language

Reply:
Done https://github.com/gravitational/teleport/pull/54019/commits/cf2c3308129f1c6d8f791c6ed8ea636b474e45e8